plugins/aws/resoto_plugin_aws/resource/ec2.py

```diff
@@ -471,6 +472,12 @@ def connect_in_graph(self, builder: GraphBuilder, source: Json) -> None:
         builder.add_edge(self, EdgeType.default, reverse=True, name=self.volume_type)
         for attachment in self.volume_attachments:
             builder.dependant_node(self, reverse=True, clazz=AwsEc2Instance, id=attachment.instance_id)
+        if self.volume_kms_key_id:
+            builder.dependant_node(
```

Comment:
should it be a reverse node?

Reply:
I don't see how the key could be the leading entity here 🤔 Any reason in particular?